### ISSUE TITLE
Melhorar a visualização nas avaliações de edições de portais

### ISSUE THREAD

**Issue by AbreuEfechou** [SpawW/WFTools#6]
O ideal seria mostrar as informações todas em uma tela, o que não tem acontecido na versão atual do script.

**Comment by SpawW** [comment]
Concordo. Será feito, seguiremos o mesmo conceito de "one page, no scrool" e tentaremos trazer as informações do FastOPR de volta para esta tela também para auxiliar no voto.
